### PR TITLE
Update Terms of Use to add a line on working with children

### PR DESCRIPTION
**Pull request by MylesJarvis**
## Context

We received confirmation that this line was necessary. I've written this PR to try and help keep capacity focussed on other continuous application work.

## Changes proposed in this pull request

Add a line into the ToU relating to working with children.

## Guidance to review

Is this suitable? It matches the screenshot in the Trello ticket. 

I've added one comment on potential duplication. This needs checking too.

## Link to Trello card

https://trello.com/c/0P8troNW/649-draft-and-add-line-to-terms-of-use-about-it-being-an-offence-to-apply-if-youve-been-barred-from-working-with-children

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: app/views/content/terms_candidate.html.erb [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
     <p class="govuk-body">You’ll be asked about convictions in the application form.</p>
```
Comment (MylesJarvis) [inline app/views/content/terms_candidate.html.erb:84]: Is this a duplicate, given that we have a line directly above it which says "You’ll be asked about any criminal convictions in the application form."

Comment (MaeveRoseveare) [reply app/views/content/terms_candidate.html.erb:84]: yeh, remove

**Review by MaeveRoseveare — APPROVED** [review]
I see we actually talk about 40 working days etc on this page, I'll dig out that Terms of use ticket and repurpose to review the entire thing to make sure we remove any content that's not applicable anymore for the new cycle.

File: app/views/content/terms_candidate.html.erb [hunk 2]
```diff
@@ -83,7 +83,7 @@
...
 
-    <p class="govuk-body">Your name will also be checked against a list of people who’ve been barred from working with children.</p>
+    <p class="govuk-body">Your name will also be checked against a list of people who’ve been barred from working with children. It's an offence to apply for a course if you’ve been barred from working with children.</p>
```
Comment (JR-G) [inline app/views/content/terms_candidate.html.erb:86]: smart quotes pls!

Comment (JR-G) [reply app/views/content/terms_candidate.html.erb:86]: ’

Comment (MylesJarvis) [reply app/views/content/terms_candidate.html.erb:86]: Ah sorry, I did one but not the second.

**Review by JR-G — APPROVED** [review]

**Pull request merged**